test/api/assignments_test.rb

```diff
@@ -112,7 +111,8 @@ def test_notify_team_members_about_submission
     team2.confirm(eve.username)
 
     Xapi.stub(:exists?, true) do
-      post '/user/assignments', {key: bob.key, code: 'THE CODE', path: 'ruby/bob/code.rb'}.to_json
+      post '/user/assignments', {key: bob.key, code: 'THE CODE', path: 'ruby/bob/code.rb','language' => 'ruby',
```

Comment:
Line is too long. [111/80]<br>Space missing after comma.<br>Space inside { missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.